### ISSUE TITLE
Incorrect npm version vs latest version - 15.12.1 is seen as 15.12.0?

### ISSUE THREAD

**Issue by JonasSonn** [lix-pm/lix.client#198]
Hi all,

We're doing some dependency vulnerability scanning for our project and is alerted that lix is out of date. 

But it seems that installing the latest version (currently: 15.12.1) is not reflected in the installed package version.

<img width="649" alt="image" src="https://github.com/lix-pm/lix.client/assets/25055574/eeab687b-71b0-437b-9649-6d38065077bb">

Best, 
Jonas

**Comment by back2dos** [comment]
Yeah, I sometimes mess up the order when publishing and then the new package is published with the old version, sorry. It's fine with  lix 16 (at elast for now ^^).
